### ISSUE TITLE
hard to tell what's wrong in case of root installed zsh

### ISSUE THREAD

**Issue by Canop** [Canop/broot#200]
This probably applies to other cases.

What was observed for a user having zsh non used and installed as root was this:

` ` ` 
Broot should be launched using a shell function (see https://github.com/Canop/broot for explanations).
The function is either missing, old or badly installed.
Can I install it now? [Y n]

Removing /home/solus/.local/share/broot/launcher/bash/1.
Writing br shell function in /home/solus/.local/share/broot/launcher/bash/1.
Removing /home/solus/.config/broot/launcher/bash/br.
Creating link from /home/solus/.config/broot/launcher/bash/br to /home/solus/.local/share/broot/launcher/bash/1.
/home/solus/.bashrc already patched, no change made.
IO Error : "Permission denied (os error 13)"
` ` ` 

and the log contained
` ` `
07:27:23 [ INFO] Starting Broot v0.13.1 with log level DEBUG
07:27:23 [DEBUG] (1) broot::shell_install: No 'installed' : we ask if we can install
07:27:27 [DEBUG] (1) broot::shell_install: proceed: true
07:27:27 [DEBUG] (1) broot::shell_install: Starting install
07:27:27 [ INFO] Writing br shell function in "/home/solus/.local/share/broot/launcher/bash/1"
07:27:27 [ INFO] Creating link from "/home/solus/.config/broot/launcher/bash/br" to "/home/solus/.local/share/broot/launcher/bash/1"
07:27:27 [DEBUG] (1) broot::shell_install::bash: considering path: "/home/solus/.bashrc"
07:27:27 [DEBUG] (1) broot::shell_install::bash: considering path: "/home/solus/.bash_profile"
07:27:27 [DEBUG] (1) broot::shell_install::bash: considering path: "/home/solus/.zshrc"
07:27:27 [DEBUG] (1) broot::shell_install::bash: considering path: "/home/solus/$ZDOTDIR/.zshrc/.zshrc"
07:27:27 [ WARN] Error: IO Error : "Permission denied (os error 13)"
07:27:27 [ INFO] bye
` ` ` 

The problem was the non accessible zsh installation.

This should be easier to diagnostic and should not block the whole installation.

**Comment by HagaiHargil** [comment]
Did you manage to solve it?

**Comment by Canop** [comment]
@HagaiHargil No, I didn't tackle it yet. I'm on a big new change introducing multi-panels and completion (in the "panels" branch) and everything else is frozen until I've finished it.

**Comment by HagaiHargil** [comment]
@Canop I completely understand, thanks for taking the time to do such a large and important refactor! 

I'm sorry if my message was hinting otherwise, I was simply trying to understand whether there's a "manual" fix I can try, since your initial report suggested that you are aware of a way to overcome it.

**Comment by Canop** [comment]
There's absolutely no problem in asking, no worry.

I don't think there's a hard problem here, other than finding the time to do a not terribly exciting change, and a rather painful one when it comes to thoroughly testing all paths.

**Comment by HagaiHargil** [comment]
I just wanted to document that if I simply run `sudo /path/to/broot/executable --install` then this issue is fixed.
